src/components/UI/Pagination/Pagination.tsx

```diff
@@ -28,12 +28,12 @@ export const Pagination: FunctionComponent<PaginationProps> = ({ totalNoOfPages,
 
   return (
     <div className="flex overflow-x-auto">
-      <div className="border border-solid border-grey-200 border-r-0 flex ml-auto items-center justify-center">
+      <div className="flex ml-auto items-center justify-center">
         <PaginationBox
           onClick={goPreviousPage}
           disable={currentPage === 1}
           data-testid="page-prev"
-          className="hover:bg-grey-100 hover:text-grey"
+          className="hover:bg-blue hover:text-white hover:rounded-md"
```

Comment:
able to update this so that when disable === true we can display 'hover:bg-grey-300', and when disable === false we display 'hover:bg-blue', so that the user will know if it is clickable a not based on the color.

Reply:
done. i added inside PaginationBox.tsx as there is someone set there when disabled is set to true.

Reply:
i put bg-grey-200 instead, 300 seems too dark.

Reply:
ok gd job!